romancal/ramp_fitting/ramp_fit_step.py

```diff
@@ -168,9 +168,18 @@ def process(self, input):
             opt_model = create_optional_results_model(input_model, opt_info)
             self.save_model(opt_model, 'fitopt', output_file=self.opt_name)
 
-        if image_info is not None:
-            out_model = create_image_model(input_model, image_info)
-            out_model.meta.cal_step.ramp_fit = 'COMPLETE'
+
+        # All pixels saturated, therefore returning an image file with zero data
+        if image_info is None:
+            # Image info order is: data, dq, var_poisson, var_rnoise, err
+            image_info = (np.zeros(input_model.data.shape[2:], dtype=input_model.data.dtype),
+                          input_model.pixeldq | input_model.groupdq[0][0] | dqflags.group['SATURATED'],
+                          np.zeros(input_model.err.shape[2:], dtype=input_model.err.dtype),
+                          np.zeros(input_model.err.shape[2:], dtype=input_model.err.dtype),
+                          np.zeros(input_model.err.shape[2:], dtype=input_model.err.dtype))
+
+        out_model = create_image_model(input_model, image_info)
+        out_model.meta.cal_step.ramp_fit = 'COMPLETE'
```

Comment:
I guess `COMPLETE` is OK here since the step ran. But may be print that `log.info` message explaining why it's a zeroed-out image? Oh, I see the message is in the exposure_pipeline code. It may be better to have it here so it can be picked up if one is running only ramp_sitting as a step.

Reply:
Yeah, I actually prefer "INCOMPLETE" (or something similar) for the ramp_fit step failure, since a ramp wasn't actually fit to it, but the PR note said to set it to "COMPLETE."

Reply:
I have added the message to the ramp_sitting step.